### PR TITLE
Ebodac 472

### PR DESCRIPTION
**Pull request by tomaszmueller**

### CODE REVIEW

**Comment by tomaszmueller** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/DisplayOfSubjectVisitInTabularFormatUiTest.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import org.junit.Before;
+import org.junit.Test;
+import org.motech.page.LoginPage;
+import org.motech.test.TestBase;
+import org.motechproject.ebodac.uitest.page.EBODACPage;
+import org.motechproject.ebodac.uitest.page.HomePage;
+import org.motechproject.ebodac.uitest.page.ParticipantEditPage;
+import org.motechproject.ebodac.uitest.page.ParticipantPage;
+
+import static junit.framework.Assert.assertTrue;
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/DisplayOfSubjectVisitInTabularFormatUiTest.java]: change it to org.junit.Assert.assertTrue;

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/DisplayOfSubjectVisitInTabularFormatUiTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    @Before
+    public void setUp() {
+        user = properties.getUserName();
+        password = properties.getPassword();
+        loginPage = new LoginPage(driver);
+        homePage = new HomePage(driver);
+        participantPage = new ParticipantPage(driver);
+        participantEditPage = new ParticipantEditPage(driver);
+        if (homePage.expectedUrlPath() != currentPage().urlPath()) {
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/DisplayOfSubjectVisitInTabularFormatUiTest.java]: you should not compare two Strings with "!=" or "==", use equals methods instead or if both strings can be null use org.apache.commons.lang.StringUtils.equals

**Comment by tomaszmueller** [comment]
@pmuchowski corrected

**Comment by tomaszmueller** [comment]
test this please

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/DisplayOfSubjectVisitInTabularFormatUiTest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    @Before
+    public void setUp() {
+        user = properties.getUserName();
+        password = properties.getPassword();
+        loginPage = new LoginPage(driver);
+        homePage = new HomePage(driver);
+        participantPage = new ParticipantPage(driver);
+        participantEditPage = new ParticipantEditPage(driver);
+        if (StringUtils.equals(homePage.expectedUrlPath(), currentPage().urlPath())) {
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/DisplayOfSubjectVisitInTabularFormatUiTest.java]: it was "!=" before and now you have equals, should not this be !StringUtils.equals?

Comment (tomaszmueller) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/DisplayOfSubjectVisitInTabularFormatUiTest.java]: right, @pmuchowski  good catch

**Comment by tomaszmueller** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

**Pull request merged**